### PR TITLE
Introduce font file option

### PR DESCRIPTION
**Pull request by Meakk**

### CODE REVIEW

**Comment by mwestphal** [comment]
This is working great ! Please add a test and doc.

File: testing/data/DATA_LICENSES.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 - Cameras.gltf: glTF-Sample-Models: Public Domain
 - cow.vtk: VTK Data: BSD
 - cow.vtp: VTK Data: BSD
+- DFLIYX1B.TTF: [fontsaddict Free license font](http://www.fontsaddict.com/font/dfliyuanxbold1b.html)
```
Comment (mwestphal) [inline testing/data/DATA_LICENSES.md:13]: Unless I'm mistaken, license in not super clear on the website and list this font as "Free for personnal use". Not ok to add in this context I'm afraid.

File: src/vtkF3DRenderer.cxx [hunk 2]
```diff
@@ -193,26 +193,47 @@ void vtkF3DRenderer::Initialize(const F3DOptions& options, const std::string& fi
...
   this->TimerActor->SetPosition(10, 10);
 
-  this->CheatSheetActor->GetTextProperty()->SetFontFamilyToCourier();
   this->CheatSheetActor->GetTextProperty()->SetFontSize(15);
   this->CheatSheetActor->GetTextProperty()->SetOpacity(0.5);
   this->CheatSheetActor->GetTextProperty()->SetBackgroundColor(0, 0, 0);
   this->CheatSheetActor->GetTextProperty()->SetBackgroundOpacity(0.5);
 
+  std::string fontPath = vtksys::SystemTools::CollapseFullPath(this->Options.FontFile);
+  std::cout << fontPath << std::endl;
```
Comment (mwestphal) [inline src/vtkF3DRenderer.cxx:213]: cout

File: README.md [hunk 3]
```diff
@@ -162,6 +162,7 @@ Options|Description
 \-\-geometry-only|For certain **full scene** file formats (gltf/glb and obj),<br>reads *only the geometry* from the file and use default scene construction instead.
 \-\-dry-run|Do not read the configuration file but consider only the command line options
 \-\-config|Read a provided configuration file instead of default one
+\-\-font-file|Use the provided FreeType compatible font file to display text
```
Comment (mwestphal) [inline README.md:165]: `UTF8 supported` or something like that.

Comment (Meakk) [reply README.md:165]: It's unrelated to utf8

Comment (mwestphal) [reply README.md:165]: What I mean is that someone looking to display unsupported char (chinese let say) may have a hard time finding out this is the option they are looking for.

File: README.md [hunk 4]
```diff
@@ -162,6 +162,7 @@ Options|Description
 \-\-geometry-only|For certain **full scene** file formats (gltf/glb and obj),<br>reads *only the geometry* from the file and use default scene construction instead.
 \-\-dry-run|Do not read the configuration file but consider only the command line options
 \-\-config|Read a provided configuration file instead of default one
+\-\-font-file|Use the provided FreeType compatible font file to display text.<br>Could be useful to display non-ASCII filenames.
```
Comment (mwestphal) [inline README.md:165]: `Can be useful`

**Review by mwestphal — APPROVED** [review]

**Pull request merged**